apps/remote_control/test/lexical/remote_control/search/indexer/extractors/module_test.exs

```diff
@@ -172,6 +172,22 @@ defmodule Lexical.RemoteControl.Search.Indexer.Extractors.ModuleTest do
       assert decorate(doc, module_ref.range) =~ "  «Some.Module».function()"
     end
 
+    test "can detect a module reference in a capture operator" do
```

Comment:
there are other cases here
  
  1. A function capture for a local public function
  2. A function capture for a local private function
  3. A function capture for an imported function
  4. A function capture for a remote function that's been aliased

Reply:
Added